js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
I'd vote for using the `usecache` flag. Like 99% of the time it will show the most up to date information. Maybe 1% of the time it will be stale.